bin/run_mdsw.sh

```diff
@@ -35,25 +36,29 @@ if [[ $# -eq 0 ]]; then
 fi
 
 mkdir "${DATADIR}" || true
+mkdir -p "${SYMBOLSCACHE}/cache" || true
+mkdir -p "${SYMBOLSCACHE}/tmp" || true
```

Comment:
I discovered this was a problem after adding `--log-file` and making sure the log file was working.